lib/humanize.rb

```diff
@@ -30,7 +30,7 @@ def humanize(locale: Humanize.config.default_locale,
   def self.for_locale(locale)
     case locale.to_sym
     # NOTE: add locales here in ealphabetical order
-    when :az, :de, :en, :es, :fr, :id, :pt, :ru
+    when :az, :de, :en, :es, :fr, :id, :pt, :ru, :ms
```

Comment:
Existing seems to be sorted alphabetically.

Reply:
on it

Reply:
done